src/services/member/repository.ts

```diff
@@ -83,7 +83,7 @@ export const MemberRepository = AppDataSource.getRepository(Member).extend({
 
   async post(data: Partial<Member> & Pick<Member, 'email'>): Promise<Member> {
     const email = data.email.toLowerCase();
-    const createdMember = await this.insert({ ...data, email });
+    const createdMember = await this.insert({ ...data, email, userAgreements: new Date() });
```

Comment:
I would add a comment that we suppose any new member accepted the agreements.

Reply:
Yes it's a really good idea !